lib/engine/db/postgres/schema.sql

```diff
@@ -0,0 +1,145 @@
+-- Cockroach db can be downloaded using following commands:
+--    wget -qO- https://binaries.cockroachdb.com/cockroach-v20.2.10.linux-amd64.tgz | tar xvz
+--    curl https://binaries.cockroachdb.com/cockroach-v20.2.10.darwin-10.9-amd64.tgz | tar -xJ
+--
+-- The following commandline starts a local node for testing:
+--    cockroach start-single-node  --insecure --listen-addr=localhost:8888 --store=type=mem,size=640MiB
+--     
+-- The following command can be use to initialize the databse for testing:
+--    cockroach sql --url postgresql://root@localhost:8888?sslmode=disable < schema.sql 
+
+drop database if exists thingengine;
+create database thingengine; 
+use thingengine;
+
+drop table if exists app ;
```

Comment:
The only tables we need are `app`, `channel`, `device`, `device_journal`. The rest can go.